### PR TITLE
feat: enable cors

### PR DESCRIPTION
**Pull request by cargopete**
Enabling CORS so we can use the graphql endpoints on the HTTP server in FE clients like GraphSeer. Because currently we're getting this errors
image

### CODE REVIEW

**Review by hopeyen — DISMISSED** [review]
not necessarily significant, but I think the previous early return and unwrap is cleaner than the change to `if {...} else {...}` with major chunk of code nested

**Review by hopeyen — DISMISSED** [review]
lgtm!

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -1,12 +1,6 @@
 name: Build and upload Docker image
 
-on:
-  release:
-    types: [published]
-  push:
-    branches: [dev]
-    tags:
-      - "*.*.*"
+on: push
```
Comment (hopeyen) [inline .github/workflows/docker.yml]: question if we should have docker built for every push...?

**Review by hopeyen — DISMISSED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (cargopete) [reply .github/workflows/docker.yml]: im debugging 😢  can't get CI to work

**Review by hopeyen — APPROVED** [review]
lets try it

File: .github/workflows/docker.yml (hunk 1 above)
Comment (hopeyen) [reply .github/workflows/docker.yml]: understood, we could set a filter to build for every push if the branch is of a certain format, like some the pattern "dev", "main", "petko_testing/***"

**Pull request merged**